gatsby/content/docs/free-small-matrix-server.mdx

```diff
@@ -0,0 +1,204 @@
+
+Open incoming for CIDR 0.0.0.0/0:
+- 22/tcp for SSH (should be open already)
+- 80/tcp for HTTP
+- 443/tcp for HTTPS
+- 8448/tcp for Matrix federation
+- 3478/tcp, 5349/tcp, 3478/udp, 5349/udp, 49152-49172/udp for TURN/STUN (you can
+  omit this if you don't run coturn for audio and video calls)
+
+### Ubuntu iptables
+
+The Oracle Cloud Ubuntu images come with somewhat restrictive iptables rules by
+default. Docker manages the instance firewall and we have the Oracle Cloud
+firewall in front, so let's remove the current firewall to avoid trouble:
+
+```sh
+apt purge netfilter-persistent iptables-persistent
+```
+
+## Remove useless stuff (optional)
+
+Oracle cloud includes a somewhat heavy monitoring daemon. We have better use for
+that memory since current versions of synapse, the Matrix homeserver, can be
+memory hungry.
+
+```sh
+snap remove oracle-cloud-agent
+apt remove snapd open-iscsi lxd lxcfs
+```
+
+## Tune server (optional)
+
+I suggest enabling swap, since there's only 1 GB of RAM.
+
+```sh
+dd if=/dev/zero of=/swap bs=1M count=1k
+chmod 0600 /swap
+mkswap /swap
+swapon /swap
+echo '/swap none swap sw 0 0' >> /etc/fstab
+```
+
+## Point domain at server
+
+Assuming you're using a new domain only for this you need the following DNS
+records:
+
+- A record `$domain` pointing to `$instance_external_ip_address`
+- CNAME record `matrix.$domain` pointing to `$domain`
+- CNAME record `riot.$domain` pointing to `$domain`
+
+
+## Use matrix-docker-ansible-deploy
+
+Follow the [guide], with some tweaks:
+
+[guide]: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/README.md
+
+### Guide in a nutshell, TL;DR
+
+```sh
+git clone https://github.com/spantaleev/matrix-docker-ansible-deploy/
+cd matrix-docker-ansible-deploy
+mkdir inventory/host_vars/matrix.$domain
+cp examples/host-vars inventory/host_vars/matrix.$domain/vars.yaml
+cp examples/hosts inventory/hosts
+$EDITOR inventory/hosts
+$EDITOR inventory/host_vars/matrix.$domain/vars.yaml
+ansible-playbook -i inventory/hosts setup.yml --tags=setup-all
+ansible-playbook -i inventory/hosts setup.yml --tags=start
+ansible-playbook -i inventory/hosts setup.yml --tags=self-check
+ansible-playbook -i inventory/hosts setup.yml -e username=$user -e password=$pass -e admin=yes --tags=register-user
+```
+
+### Serve the base domain as well (unless you already have something serving it)
+
+```yaml
+matrix_nginx_proxy_base_domain_serving_enabled: true
+```
+
+### Disable all extras because of RAM
+
+```yaml
+matrix_dimension_enabled: false
+matrix_mxisd_enabled: false
+matrix_mailer_enabled: false
+matrix_coturn_enabled: true # disable to save more ram, affects audio/video
+```
+
+### Configure the public IP address for coturn
+
+```yaml
+matrix_coturn_turn_external_ip_address: $instance_external_ip_address
+```
+
+### Limit rooms because of limited RAM
+
+```yaml
+matrix_synapse_configuration_extension_yaml: |
+  limit_remote_rooms:
+    enabled: true
+    complexity: 1.0
```

Comment:
This is quite a low value. it's fine to leave this, but you should explain what it will affect (joining large rooms) and that it can be tweaked later if necessary.

Reply:
are there any guidelines for suitable values?